src/workloads/transactions/LLTMixed.yml

```diff
@@ -561,11 +561,15 @@ Actors:
       Duration: *SnapshotScannerLongDuration
       ScanDuration: *SnapshotScannerLongDuration
 
-#AutoRun:
-#  - When:
-#      mongodb_setup:
-#        $eq:
-#          - atlas
-#          - replica
-#          - replica-all-feature-flags
-#          - single-replica
+AutoRun:
+  - When:
+      mongodb_setup:
+        $eq:
+          - atlas
```

Comment:
Do you want atlas, atlas-like-replica, or both? Any of those are fine, just double checking intention.

Reply:
good point, I should probably add  atlas-like-replica too.